Merge master into live-based

OPSTestProduction opened the pull request.

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

azclibot committed:
azure-cli release notes 2.0.79

e2ebd5 commented:
Docs Build status updates of commit _[15cf401](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/commits/15cf40117a8ea7fe39bd95e4161a17e80fc88182)_: 

:clock10: **Full build**: the average full build time is _7 min(s) 3 sec(s)_, this is based on the last _61_ build(s) for this repository.

e2ebd5 commented:
Docs Build status updates of commit _[15cf401](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/commits/15cf40117a8ea7fe39bd95e4161a17e80fc88182)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/authenticate-azure-cli.md](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/docs-ref-conceptual/authenticate-azure-cli.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/authenticate-azure-cli?view=azure-cli-latest&branch=pr-en-us-85) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/authenticate-azure-cli?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile)<br/> [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/authenticate-azure-cli?view=azure-cli-2018-03-01-hybrid&branch=pr-en-us-85) (azure-cli-2018-03-01-hybrid) | [Details](#user-content-a70f22a3-711f71a5)
[docs-ref-conceptual/azure-cli-configuration.md](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/docs-ref-conceptual/azure-cli-configuration.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/azure-cli-configuration?view=azure-cli-latest&branch=pr-en-us-85) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/azure-cli-configuration?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile)<br/> [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/azure-cli-configuration?view=azure-cli-2018-03-01-hybrid&branch=pr-en-us-85) (azure-cli-2018-03-01-hybrid) | [Details](#user-content-8219011c-711f71a5)
[docs-ref-conceptual/create-an-azure-service-principal-azure-cli.md](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/docs-ref-conceptual/create-an-azure-service-principal-azure-cli.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/create-an-azure-service-principal-azure-cli?view=azure-cli-latest&branch=pr-en-us-85) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/create-an-azure-service-principal-azure-cli?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile)<br/> [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/create-an-azure-service-principal-azure-cli?view=azure-cli-2018-03-01-hybrid&branch=pr-en-us-85) (azure-cli-2018-03-01-hybrid) | [Details](#user-content-876bcc5f-711f71a5)
[.openpublishing.publish.config.json](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[2017-03-09-profile/docs-ref-autogen/TOC.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/security/task?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/account.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/account.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/account?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/app.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/app.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/app?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/app/credential.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/app/credential.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/app/credential?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/app/owner.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/app/owner.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/app/owner?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/app/permission.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/app/permission.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/app/permission?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/group.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/group.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/group?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/group/member.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/group/member.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/group/member?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/group/owner.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/group/owner.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/group/owner?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/signed-in-user.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/signed-in-user.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/signed-in-user?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/sp.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/sp.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/sp?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/sp/credential.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/sp/credential.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/sp/credential?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/sp/owner.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/sp/owner.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/sp/owner?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/user.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/user.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/user?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/appconfig.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/appconfig.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/appconfig?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/appconfig/credential.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/appconfig/credential.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/appconfig/credential?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/appconfig/feature.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/appconfig/feature.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/appconfig/feature?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/appconfig/feature/filter.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/appconfig/feature/filter.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/appconfig/feature/filter?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/appconfig/kv.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/appconfig/kv.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/appconfig/kv?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/appconfig/revision.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/appconfig/revision.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/appconfig/revision?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/artifacts.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/artifacts.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/artifacts?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/artifacts/universal.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/artifacts/universal.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/artifacts/universal?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/boards.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/boards.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/boards?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/boards/area.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/boards/area.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/boards/area?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/boards/area/project.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/boards/area/project.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/boards/area/project?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/boards/area/team.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/boards/area/team.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/boards/area/team?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/boards/iteration.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/boards/iteration.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/boards/iteration?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/boards/iteration/project.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/boards/iteration/project.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/boards/iteration/project?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/boards/iteration/team.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/boards/iteration/team.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/boards/iteration/team?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/boards/work-item.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/boards/work-item.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/boards/work-item?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/boards/work-item/relation.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/boards/work-item/relation.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/boards/work-item/relation?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/cloud.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/cloud.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/cloud?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/devops.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/devops.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/devops?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/devops/admin/banner.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/devops/admin/banner.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/devops/admin/banner?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/devops/extension.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/devops/extension.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/devops/extension?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/devops/project.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/devops/project.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/devops/project?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/devops/security.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/devops/security.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/devops/security?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/devops/security/group.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/devops/security/group.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/devops/security/group?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/devops/security/group/membership.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/devops/security/group/membership.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/devops/security/group/membership?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/devops/security/permission.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/devops/security/permission.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/devops/security/permission?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/devops/security/permission/namespace.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/devops/security/permission/namespace.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/devops/security/permission/namespace?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/devops/service-endpoint.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/devops/service-endpoint.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/devops/service-endpoint?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/devops/service-endpoint/azurerm.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/devops/service-endpoint/azurerm.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/devops/service-endpoint/azurerm?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/devops/service-endpoint/github.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/devops/service-endpoint/github.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/devops/service-endpoint/github?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/devops/team.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/devops/team.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/devops/team?view=azure-cli-2017-03-09-profile&branch=pr-en-us-85) (azure-cli-2017-03-09-profile) |

This comment lists only the first 50 files in the pull request.
<a id="a70f22a3-711f71a5"></a>
### [docs-ref-conceptual/authenticate-azure-cli.md](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/docs-ref-conceptual/authenticate-azure-cli.md)
 - **[Warning]** `````Missing attribute: either ms.prod or ms.service is required. Use ms.prod for on-premise products, or ms.service for cloud services.`````
 - **[Warning]** `````Missing attribute: ms.prod. If you specify ms.technology, you must also specify ms.prod. `````
 - **[Warning]** `````Deprecated attribute: ms.component. Use ms.subservice instead. `````
 - **[Warning]** `````Missing attribute: either ms.prod or ms.service is required. Use ms.prod for on-premise products, or ms.service for cloud services.`````
 - **[Warning]** `````Missing attribute: ms.prod. If you specify ms.technology, you must also specify ms.prod. `````
 - **[Warning]** `````Deprecated attribute: ms.component. Use ms.subservice instead. `````
 - **[Warning]** `````Missing attribute: either ms.prod or ms.service is required. Use ms.prod for on-premise products, or ms.service for cloud services.`````
 - **[Warning]** `````Missing attribute: ms.prod. If you specify ms.technology, you must also specify ms.prod. `````
 - **[Warning]** `````Deprecated attribute: ms.component. Use ms.subservice instead. `````

<a id="8219011c-711f71a5"></a>
### [docs-ref-conceptual/azure-cli-configuration.md](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/docs-ref-conceptual/azure-cli-configuration.md)
 - **Line 92**: **[Warning]** `````Link 'http://aka.ms/AzureCliLegal' is insecure. Links to Microsoft sites must use 'https' instead of 'http'.`````
 - **Line 92**: **[Warning]** `````Link 'http://aka.ms/AzureCliLegal' is insecure. Links to Microsoft sites must use 'https' instead of 'http'.`````
 - **Line 92**: **[Warning]** `````Link 'http://aka.ms/AzureCliLegal' is insecure. Links to Microsoft sites must use 'https' instead of 'http'.`````

<a id="876bcc5f-711f71a5"></a>
### [docs-ref-conceptual/create-an-azure-service-principal-azure-cli.md](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/docs-ref-conceptual/create-an-azure-service-principal-azure-cli.md)
 - **[Warning]** `````Missing attribute: either ms.prod or ms.service is required. Use ms.prod for on-premise products, or ms.service for cloud services.`````
 - **[Warning]** `````Missing attribute: ms.prod. If you specify ms.technology, you must also specify ms.prod. `````
 - **[Warning]** `````Missing attribute: either ms.prod or ms.service is required. Use ms.prod for on-premise products, or ms.service for cloud services.`````
 - **[Warning]** `````Missing attribute: ms.prod. If you specify ms.technology, you must also specify ms.prod. `````
 - **[Warning]** `````Missing attribute: either ms.prod or ms.service is required. Use ms.prod for on-premise products, or ms.service for cloud services.`````
 - **[Warning]** `````Missing attribute: ms.prod. If you specify ms.technology, you must also specify ms.prod. `````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C6%5C0762c6d6-cb91-510a-10b9-a881d0bb3e83%5CPullRequest%5C202001060801103041-85%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=lPEh0KEgnTTFG2CsHzSBB1wCc2LvUEFzGDv0EZc%2BOdQ%3D&st=2020-01-06T08%3A04%3A56Z&se=2020-02-06T08%3A09%3A56Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

Commit:
CI Update

jsntcy committed:
Merge pull request #1731 from azclibot/cli-release-notes-2.0.79

release note for azure cli 2.0.79

jsntcy committed:
Merge pull request #1737 from MicrosoftDocs/master

Merge master to live for 2.0.79

Commit:
CI Update

jsntcy committed:
update author and date

jsntcy committed:
fix ms.author

jsntcy committed:
Merge pull request #1738 from MicrosoftDocs/master

merge master to live

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

bim-msft committed:
Add HDInsight release notes (#1739)

* Add HDInsight release notes

* Update alias

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

azclibot committed:
release note for azure cli 2.0.80 (#1748)

* azure-cli release notes 2.0.80

* update ms.date

* Update release-notes-azure-cli.md

* fix typo

fix typo

Co-authored-by: Yu Chen <ychenu@microsoft.com>

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

jsntcy committed:
merge master to live for Azure CLI 2.0.80 (#1750)

* CI Update

* CI Update

* Add HDInsight release notes (#1739)

* Add HDInsight release notes

* Update alias

* CI Update

* CI Update

* CI Update

* CI Update

* CI Update

* CI Update

* CI Update

* release note for azure cli 2.0.80 (#1748)

* azure-cli release notes 2.0.80

* update ms.date

* Update release-notes-azure-cli.md

* fix typo

fix typo

Co-authored-by: Yu Chen <ychenu@microsoft.com>

* CI Update

Co-authored-by: VSC-Service-Account <vscavu@microsoft.com>
Co-authored-by: Bin Ma <bim@microsoft.com>
Co-authored-by: Azure CLI Bot <azclibot@microsoft.com>

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Taojunshen committed:
Merge branch 'live' into master

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

BhargaviAnnadevara committed:
(AzureCXP) Fixes MicrosoftDocs/azure-docs-cli#1774

Fixes #1707 and #1774.

barbkess committed:
Merge pull request #1781 from BhargaviAnnadevara-MSFT/patch-1

(AzureCXP) Fixes MicrosoftDocs/azure-docs-cli#1774

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
final update of TOC, new overview, and minor updates to getting started and install.

dbradish-microsoft committed:
Update TOC.yml

dbradish-microsoft committed:
Update get-started-with-azure-cli.md

dbradish-microsoft committed:
Update what-is-azure-cli.md

dbradish-microsoft committed:
Update install-azure-cli.md

dbradish-microsoft committed:
Update TOC.yml

dbradish-microsoft committed:
Update install-azure-cli.md

barbkess committed:
Merge pull request #1784 from dbradish-microsoft/dbradish_2020-01-30

final update of TOC, new overview, and minor updates to getting start…

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

azclibot committed:
release note for azure cli 2.0.81 (#1787)

* azure-cli release notes 2.0.81

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

azclibot committed:
Update CLI extensions available doc.
 Triggered by Azure/azure-cli-extensions - TRAVIS_BUILD_ID=646604515
https://github.com/Azure/azure-cli-extensions/commit/e60e656934da37db38da7dd562ba1fd673776396

jsntcy committed:
Merge pull request #1795 from MicrosoftDocs/az-ext-list-ci-646604515

Update CLI extensions available doc

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

Commit:
CI Update

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)

azclibot committed:
azure-cli release notes 2.1.0

jsntcy committed:
Merge pull request #1805 from azclibot/cli-release-notes-2.1.0

release note for azure cli 2.1.0

fenxu committed:
Merging changes synced from https://github.com/MicrosoftDocs/azure-docs-cli (branch master)
